kod/util/questengine.kod

```diff
@@ -7393,11 +7410,12 @@ CheckQuestDeadlines()
       return;
    }
 
-   Jumpstart()
-   "A quick and dirty way to get things running again."
+   Reset()
+   "Resets the quest engine: clears and rebuilds quests and recreates the quest nodes."
    {
-      send(self,@Recreate);
-      send(self,@RecreateQuestNodes);
+      Send(self,@ClearAllQuests);
```

Comment:
If we don't do this, new quests and quests nodes will be called and added to monsters' `plActiveQuestNode` list. Called enough times, this will result in a `plActiveQuestNode` that far exceeds its intended size.

This is a portion of the callstack that occurs when calling jumpstart: for every quest, `Init` is called and subsequently so is `QuestNode::Init`:
![image](https://github.com/user-attachments/assets/c92fb76c-fc82-46e6-b429-e5ca136888e7)